cloudinary_cli/utils/api_utils.py

```diff
@@ -48,6 +48,22 @@ def query_cld_folder(folder):
     return files
 
 
+def regen_derived_version(public_id, options):
```

Comment:
This function doesn't really do what it says.

It's a wrapper around `uploader.explicit`.
anyone can call this function with any option and it will just do whatever is passed in the options.

You can make the function more specific and pass only the options that are needed for regen. 

Or make it more generic and pass the message as well.

Reply:
So I've made the function specific to regen now by passing only the required params